specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-05-01/sparkConfiguration.json

```diff
@@ -0,0 +1,214 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-05-01",
+    "title": "SynapseManagementClient",
+    "description": "Azure Synapse Analytics APIs"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sparkconfigurations/{sparkConfigurationName}": {
```

Comment:
This new resource type only has a GET exposed.  How is it created?

Reply:
Yes, We need just GET operations here to attach spark configurations while updationg the SparkPool